Assets/MixedRealityToolkit/Input/Scripts/InputSources/GamepadInput.cs

```diff
@@ -89,7 +90,7 @@ private void Update()
         private void HandleGamepadAPressed()
         {
             // TODO: Should this handle Submit from Edit > ProjectSettings > Input ?
-            if (Input.GetButtonDown(GamePadButtonA))
+            if (UnityEngine.Input.GetButtonDown(GamePadButtonA))
```

Comment:
We should reconsider the `MixedRealityToolkit.Input` namespace, to not hide `UnityEngine.Input` in projects that import the toolkit.

Reply:
Agreed, I think leaving it the same as it is, should be okay. eg: `InputModule`

Reply:
Agreed. Will rename folder and namespace to InputModule

Reply:
Resolved

Reply:
With the namespace rename, can all of the `UnityEngine.` additions be removed?

Reply:
Should be able to

Reply:
Sure, can re-rename them.  Although in some way it's good as it shows it's coming from Unity not MRTK.  Although I was thinking this last night.

Reply:
Renamed